src/client.ts

```diff
@@ -70,6 +72,13 @@ export class DefaultClient implements Client {
     return this.getApi().then(api => api.previewSession(token, linkResolver, defaultUrl, cb));
   }
 
+  getPreviewResolver(token: string, documentId?: string): PreviewResolver {
```

Comment:
What the purpose of having this function here ?
We have the same in the ResolvedApi, so for which purpose would we need this one ?